tests/ReflectionToolsTest.php

```diff
@@ -19,6 +19,85 @@ public function testGetMethodsDoesNotReturnStaticMethods()
         $this->assertCount(0, $methods);
     }
 
+    public function testGetReflectionFunction()
+    {
+        $reflectionFunc = function()
+        {
+            return 'reflectedFunction';
+        };
+        $functions = (new ReflectionTools)->getReflectionFunction($reflectionFunc);
+
+        $this->assertCount(0, $functions->getParameters());
```

Comment:
It would actually be better to check that `$function` (without `s`, please) is an instance of `ReflectionFunction` and that `getName()` returns the proper value!